libraries/AdaptiveExpressions/ExpressionProperties/ExpressionProperty.cs

```diff
@@ -29,11 +29,22 @@ public ExpressionProperty()
         /// <param name="value">An object containing the value to be set.</param>
         public ExpressionProperty(object value)
         {
+            OriginalValue = value;
 #pragma warning disable CA2214 // Do not call overridable methods in constructors (fixing this would require further redesign of this class and derived types, excluding it for now).
             SetValue(value);
 #pragma warning restore CA2214 // Do not call overridable methods in constructors
         }
 
+        /// <summary>
+        /// Gets or sets the original value of the expression property.
+        /// </summary>
+        /// <value>
+        /// The original value.
+        /// </value>
+#pragma warning disable CA1721 // Property names should not match get methods (by design and we can't change it because of binary compat)
```

Comment:
Can we confirm do we really need this property? If so, do we need this in the base class or should we add it into the ValueExpression class?